client/ratings_reviews/helpers.js

```diff
@@ -122,12 +122,43 @@ const sortReviews = (reviews, option) => {
   return reviews;
 };
 
+const applyStarFilters = (reviews, starFilters) => {
+  if (starFilters.length === 0) { return reviews; }
+  var output = [];
+  for (var i = 0; i < reviews.length; i++) {
+    if (starFilters.indexOf(reviews[i].rating) !== -1) {
+      output.push(reviews[i]);
+    }
+  }
+  return output;
+};
+
+const formatReviewTile = (summary, body, photos) => {
+  var additionalBody = '';
+  var showAdditionalBodyButton = false;
+  var showPhotos = false;
+  if (summary.length > 60) {
+    summary = summary.slice(0, 61) + '...';
+  }
+  if (body.length > 250) {
+    additionalBody = body.slice();
+    body = body.slice(0, 251) + '...';
+    showAdditionalBodyButton = true;
+  }
+  if (photos.length !== 0) {
+    showPhotos = true;
+  }
+  return [summary, body, additionalBody, showAdditionalBodyButton, showPhotos];
+};
+
 module.exports = {
```

Comment:
For a future optimization, I think you can just list the function names here (comma separated) instead of doing the 'key-value pair' format. Just to make the code a bit cleaner, I think the format you're using now is mostly helpful if you need the function to be accessed with a different name than what it has in this helpers file.

Reply:
Neat! I didn't know this before, thank you so much for pointing this out! I will keep this in mind for future optimization.